certified-connectors/RAPID Platform/apiProperties.json

```diff
@@ -0,0 +1,65 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "tenantName": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify name of your tenant",
+          "displayName": "Tenant Name",
+          "tooltip": "Provide your tenant name"
+        }
+      },
+      "siteName": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify name of your Site",
+          "displayName": "Site Name",
+          "tooltip": "Provide your site name"
+        }
+      },
+      "token": {
+        "oAuthSettings": {
+          "clientId": "b427679a-323d-4beb-b679-fa1efa0eed89",
```

Comment:
@MattFranklin-Rapid You can replace the client Id values with dummy values like below:

` "clientId": "[[DUMMY]]",`

Reply:
Updated specifically this clientID key to [[DUMMY]] but haven't been able to find any documentation relating to how this functions. I'm assuming only this key is needed to be changed but am not sure.